projects/js-packages/charts/src/components/tooltip/base-tooltip.module.scss

```diff
@@ -9,11 +9,3 @@
   pointer-events: none;
   transform: translate(-50%, -100%);
 }
-
-:global(.visx-tooltip-glyph) {
```

Comment:
Moved this to `line-chart.module.scss` because we're not using base-tooltip in bar chart component anymore. Adding this global style in `line-chart.module` also applies to bar chart.